src/index.js

```diff
@@ -13,6 +13,16 @@ async function main() {
 
   await initDb();
 
+  app.use(async (ctx, next) => {
+    await next();
+    if (!ctx.body && ctx.status === 404) {
```

Comment:
what about other errors?

Reply:
It's not a global error handler middleware, it is needed only for separate 404 not found route errors from 404 not found in DB errors. All the other errors are thrown through a ctx.throw and have no need in any global handler.